examples/layers/JSONLayers/WORLD_DTM.json

```diff
@@ -8,7 +8,8 @@
     "updateStrategy": {
         "type": 1,
         "options": {
-            "groups": [3, 7, 9]
+            "groups": [3, 7, 9],
+            "forceLoadingStep": true
```

Comment:
It seems you added `forceLoadingStep = true` for all layers using the Group strategy.
Could it still make sense to use the Group stragegy without `forceLoadingStep`?

Reply:
completely agree